ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml

```diff
@@ -60,9 +60,21 @@ tests:
     - as: install
       cli: latest
       commands: |
-        oc create namespace obo
-        operator-sdk run bundle -n obo "$OO_BUNDLE"
-        oc wait --for condition=Available -n obo deployment observability-operator
+        oc create namespace coo
+        operator-sdk run bundle -n coo "$OO_BUNDLE"
+        if [ -e "${SHARED_DIR}/coo-csv.yml" ]; then
+          rm -f "${SHARED_DIR}/coo-csv.yml"
+        fi
+        tries=30
+        while [[ $tries -gt 0 ]] && ! oc -n coo rollout status deploy/obo-prometheus-operator; do
+          sleep 10
+          ((tries--))
+        done
+        oc wait -n coo --for=condition=Available deploy/obo-prometheus-operator --timeout=300s
+        oc wait -n coo --for=condition=Available deploy/obo-prometheus-operator-admission-webhook --timeout=300s
+        oc wait -n coo --for=condition=Available deploy/observability-operator --timeout=300s
+        CSV_NAME=$(oc -n coo get csv | grep 'observability-operator' | awk '{print $1}')
+        oc -n coo get csv "$CSV_NAME" -oyaml > "${SHARED_DIR}/coo-csv.yml"
```

Comment:
I have no doubts this works.
Have you explored editing the CSV before the bundle is build? I don't know the details but I assume there is a build step where we build the bundle from the merge PR. If we edit the CSV to add our cli arg there, we don't have to wait for the operator to become available and then change it. We could also add a `kustomize` step on obo specifically for this.

Reply:
From ci-operator doc, we can see that images can be substituted when build the bundle andhave no way to edit the csv for now, also checked with ci-operator team for the issue https://redhat-internal.slack.com/archives/CBN38N3MW/p1727230485893919

Reply:
Thanks for checking on this!